### DISCUSSION TITLE
[QUESTION]:重启服务时，如何从文件开始读取数据，而不是从上次读取的偏移处开始？在跑小demo时，重启服务，无法重新从文件开始处读取。

### DISCUSSION THREAD

**Discussion by yyuanl** [alibaba/ilogtail#365] [Help] [answered]
Before asking a question, please first consider:

- Searching [existing discussions](https://github.com/alibaba/ilogtail/discussions/)
- Searching [existing issues](https://github.com/alibaba/ilogtail/issues/)
- Searching [English Doc](https://github.com/alibaba/ilogtail/tree/main/docs/en)
- Searching Google

-------

提问之前，建议先在 iLogtail 社区寻找答案。

- 查看 [存在的 discussions](https://github.com/alibaba/ilogtail/discussions/)
- 查看 [存在的 issues](https://github.com/alibaba/ilogtail/issues/)
- 查看 [中文文档](https://ilogtail.gitbook.io/ilogtail-docs/about/readme)
- 查看 Google

如果仍然无法找到答案，我们建议您在[discussions](https://github.com/alibaba/ilogtail/discussions/)进行提问。

**Comment by messixukejia** [comment, marked as answer]
如果是出于测试目的，清空checkpoint/就可以了。

**Comment by messixukejia** [comment]
问题交流建议到discussion中吧，我这里切换下。
